test/benchmarks/CMakeLists.txt

```diff
@@ -41,3 +42,43 @@ if(ZLIB_ENABLE_TESTS)
     add_test(NAME benchmark_zlib
         COMMAND ${CMAKE_CROSSCOMPILING_EMULATOR} $<TARGET_FILE:benchmark_zlib>)
 endif()
+
+if (WITH_BENCHMARK_APPS)
+    option(BUILD_ALT_BENCH "Link against alternative zlib implementation" OFF)
+
+    # Search for libpng package
+    find_package(PNG QUIET REQUIRED)
```

Comment:
Have you thought about using `FetchContent` to download and compile `libpng`?

Reply:
I guess I could in the even that libpng is not found.  I suspect it's basically what google benchmark is doing?

Reply:
Yeah. And at least then it can be compiled on Windows.